.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
       - master
 
 env:
-  POETRY_VERSION: "1.3.0"
+  POETRY_VERSION: "1.6.1"
```

Comment:
I'm trying the latest Poetry in 16f383214038989b7a5e9271cf76d803b7a5e948, but can simply revert if it's unwanted.